mos_tests/sahara/sahara_tests.py

```diff
@@ -0,0 +1,74 @@
+#    Copyright 2016 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import os
```

Comment:
We should import things in next order (and by alphabet into groups of imports):
standard library imports
"new line"
third-party library imports 
"new line"
application imports

in this case we want to have:
import logging
import os  - because L < O (string comparison)

import pytest

from mos_tests.functions.base import OpenStackTestCase


logger = logging.getLogger(__name__) - there is 2 new lines before this declaration